package.json

```diff
@@ -17,7 +17,7 @@
     "compile": "tsc",
     "lint": "yarn tslint && yarn prettier",
     "lint:fix": "yarn tslint --fix && yarn prettier --write",
-    "clean": "yarn gatsby clean && rm -rf elm-stuff storybook/public",
+    "clean": "lerna run clean && yarn gatsby clean && rm -rf elm-stuff storybook/public",
```

Comment:
Should we add a script similar to big-frontend-repo where it tells you to clean if you're running storybook and the JS artefacts exist

Reply:
Issue to resolve: https://github.com/cultureamp/kaizen-design-system/issues/82